internal/pkg/cli/job_deploy.go

```diff
@@ -168,9 +179,18 @@ func (o *deployJobOpts) Execute() error {
 			Tags:        tags.Merge(o.targetApp.Tags, o.resourceTags),
 		},
 		Options: deploy.Options{
-			ForceNewUpdate: o.forceNewUpdate,
+			DisableRollback: o.disableRollback,
 		},
 	}); err != nil {
+		if o.disableRollback {
+			stackName := stack.NameForService(o.targetApp.Name, o.targetEnv.Name, o.name)
+			rollbackCmd := fmt.Sprintf("aws cloudformation rollback-stack --stack-name %s --role-arn %s", stackName, o.targetEnv.ExecutionRoleARN)
+			log.Infof(`It seems like you have disabled automatic stack rollback for this deployment. To debug, you can visit the AWS console to inspect the errors.
```

Comment:
are we printing this log statement twice, can we remove the one at l. 164?

Reply:
Thanks for catching this!! It was a big typo :homer-disappearing